onnxruntime/python/tools/microbench/softmax.py

```diff
@@ -0,0 +1,58 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import argparse
+from dataclasses import dataclass
+
+import numpy as np
+from benchmark import BenchmarkOp, add_arguments
+
+
+@dataclass
+class OpParam:
+    dim1: int
+    dim2: int
+    dim3: int
+
+    data_type: type
+
+
+class BenchmarkSoftmax(BenchmarkOp):
+    def __init__(self, args):
+        BenchmarkOp.__init__(self, args)
+
+    @classmethod
+    def create_inputs_outputs(cls, op_param):
+        np.random.seed(0)
+        a = np.random.rand(op_param.dim1, op_param.dim2, op_param.dim3).astype(op_param.data_type)
```

Comment:
suggest using meaningful names instead of a, b, c, d. Likewise in `topk.py`.

Reply:
Done.